types/v56/types_test.go

```diff
@@ -0,0 +1,229 @@
+package types
+
+import (
+	"encoding/xml"
+	"testing"
+)
+
+func TestAdminVDCResourcePoolSerialization(t *testing.T) {
```

Comment:
This test is not needed as doesn't validate real VCD requests and can't be run with different Vcd versions.